### PR TITLE
fix: filter custom attrs

### PR DESCRIPTION
**Pull request by nirgur**

### CODE REVIEW

File: packages/user-management-widget/src/lib/widget/mixins/initMixin/initComponentsMixins/initUsersTableMixin.ts [hunk 1]
```diff
@@ -30,6 +66,7 @@ export const initUsersTableMixin = createSingletonMixin(
             e.detail.value.map(({ loginIds }) => loginIds),
           );
         });
+        this.#setCustomRenderer();
```
Comment (dorsha) [inline packages/user-management-widget/src/lib/widget/mixins/initMixin/initComponentsMixins/initUsersTableMixin.ts:69]: Do we still need this?
image

Comment (nirgur) [reply packages/user-management-widget/src/lib/widget/mixins/initMixin/initComponentsMixins/initUsersTableMixin.ts:76]: Yes, we do

File: packages/user-management-widget/e2e/user-management-widget.spec.ts [hunk 2]
```diff
@@ -186,11 +221,16 @@ test.describe('widget', () => {
...
     expect(removePasskeyTrigger).toBeDisabled();
 
-    // wait for widget state
-    await page.waitForTimeout(STATE_TIMEOUT);
-
     // select non-editable user (editable: false)
-    await page.locator('descope-checkbox').nth(4).click();
+    const NonEditableUserCheckbox = await getTableBodyCellContentLocatorByIndex(
+      page,
+      3,
```
Comment (dorsha) [inline packages/user-management-widget/e2e/user-management-widget.spec.ts:227]: Why 3 and not 4? before it was nth 4...

File: packages/user-management-widget/e2e/user-management-widget.spec.ts [hunk 3]
```diff
@@ -215,10 +260,15 @@ test.describe('widget', () => {
...
 
     // de-select enabled and editable user
-    await page.locator('descope-checkbox').nth(1).click();
+    await editableUserCheckbox.click();
 
     // select disabled and editable user
-    await page.locator('descope-checkbox').nth(2).click();
+    const disabledUserCheckbox = await getTableBodyCellContentLocatorByIndex(
+      page,
+      1,
```
Comment (dorsha) [inline packages/user-management-widget/e2e/user-management-widget.spec.ts:268]: Why 1 and not 2? before it was nth 2...

**Review by dorsha — COMMENTED** [review]
Two questions

**Review by dorsha — APPROVED** [review]

**Pull request merged**